sql/country_statistics_working.sql

```diff
@@ -0,0 +1,61 @@
+CREATE MATERIALIZED VIEW country_statistics AS
+  WITH country_counts AS (
+    SELECT cc.changeset_id,
+        countries.id AS country_id,
+        countries.name AS country_name,
+        cc.edit_count
+      FROM (changesets_countries cc
+        JOIN countries ON ((cc.country_id = countries.id)))
+    ), agg_stats AS (
+    SELECT cc.country_id,
+        cc.country_name,
+        array_agg(chg.id) AS changesets,
```

Comment:
Let's drop this; it bloats the results tremendously.

Reply:
Oversight.  Fixed.